gammapy/scripts/make_bg_cube_models.py

```diff
@@ -0,0 +1,329 @@
+    to their altitude and azimuth angle.
+    """
+    if DEBUG:
+        print()
+        print("###############################")
+        print("# Starting group_observations #")
+        print("###############################")
+
+    # read bg observation table from file
+    indir = os.environ['PWD'] + '/'
+    infile = indir + 'bg_observation_table.fits.gz'
+    observation_table = ObservationTable.read(infile)
+
+    # split observation table according to binning
+
+    # define a binning in altitude angle
+    # TODO: ObservationGroups
+    # https://github.com/mapazarr/gammapy/blob/bg-api/dev/background-api.py#L55
+    if DEBUG:
+        print()
+        print("altitude bin boundaries")
+        print(repr(altitude_edges))
+
+    # define a binning in azimuth angle
+    # TODO: ObservationGroups
+    # https://github.com/mapazarr/gammapy/blob/bg-api/dev/background-api.py#L55
+    if DEBUG:
+        print()
+        print("azimuth bin boundaries")
+        print(repr(azimuth_edges))
+
+    # wrap azimuth angles to (-90, 270) deg
+    # TODO: needs re-thinking if azimuth angle definitions change!!!
+    #       or if user-defined azimuth angle bins are allowed!!!
+    azimuth = Angle(observation_table['AZ_PNT']).wrap_at(Angle(270., 'degree'))
+    observation_table['AZ_PNT'] = azimuth
+
+    # create output folder if not existing
+    outdir = os.environ['PWD'] + '/splitted_obs_list/'
+    if not os.path.isdir(outdir):
+        os.mkdir(outdir)
+    else:
+        # clean folder if available
+        for oldfile in os.listdir(outdir):
+            os.remove(outdir + oldfile)
+
+    # loop over altitude and azimuth angle bins: remember 1 bin less than bin boundaries
+    for i_alt in range(len(altitude_edges) - 1):
+        if DEBUG:
+            print()
+            print("bin alt", i_alt)
+        for i_az in range(len(azimuth_edges) - 1):
+            if DEBUG:
+                print()
+                print("bin az", i_az)
+
+            # filter observation table
+            observation_table_filtered = observation_table
+
+            selection = dict(type='par_box', variable='ALT_PNT',
+                             value_range=(altitude_edges[i_alt], altitude_edges[i_alt + 1]))
+            observation_table_filtered = observation_table_filtered.select_observations(selection)
+
+            selection = dict(type='par_box', variable='AZ_PNT',
+                             value_range=(azimuth_edges[i_az], azimuth_edges[i_az + 1]))
+            observation_table_filtered = observation_table_filtered.select_observations(selection)
+
+            if DEBUG:
+                print(observation_table_filtered)
+
+            # skip bins with no obs
+            if len(observation_table_filtered) == 0:
+                continue # skip the rest
+
+            # save the observation list to a fits file
+            outfile = outdir +\
+                     'bg_observation_table_alt{0}_az{1}.fits.gz'.format(i_alt, i_az)
```

Comment:
Done. It's hard though to compare models. See workaround in  `examples/plot_bg_cube_model_comparison.py` (included in this PR). Lines ~50 - ~70.